tests/attr/test_common.py

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env python3
 from __future__ import print_function
```

Comment:
should we remove the `__future__` statements then? The future is now :)  ditto to all the files in the tests directory.

Reply:
Sure, good point, will remove them!